src/pykiso/lib/connectors/cc_grpc_client.py

```diff
@@ -0,0 +1,216 @@
+##########################################################################
+# Copyright (c) 2023-2023 Accenture GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+Communication Channel Via GRPC
+********************************
+
+:module: cc_grpc_client
+
+:synopsis: Grpc communication channel
+
+.. currentmodule:: cc_grpc_client
+
+list of todo items:
+* Extend by supporting a path instead of files so that many proto files can be used
+* Extend with security capabilities
+
+"""
+
+import importlib.util
+import inspect
+import logging
+from pathlib import Path
+from typing import Dict, Optional
+
+import grpc
+
+from pykiso import connector
+
+log = logging.getLogger(__name__)
+
+
+class GrpcClient(connector.CChannel):
```

Comment:
Not sure if this is an unwritten pykiso rule (:
```suggestion
class CCGrpcClient(connector.CChannel):
```